letta/client/client.py

```diff
@@ -152,10 +152,11 @@ def send_message(
         stream: Optional[bool] = False,
         stream_steps: bool = False,
         stream_tokens: bool = False,
+        image: Optional[str] = None,
```

Comment:
I think the ideal way to do this is we actually make `message` (currently `str`) be type `MultiMediaContent`, which itself is `Union[str, List[MultiMediaContentPart]]`

Then we will be able to support multiple images and future modalities like audio.

Reply:
sounds good, updated. see message in your other comment